docs/examples/use_cases/pytorch/efficientnet/image_classification/dataloaders.py

```diff
@@ -309,6 +316,58 @@ def __len__(self):
         return len(self.dataloader)
 
 
+class DALIProxyWrapper(object):
```

Comment:
Wouldn't the user be scared once they see such a complex wrapper for proxy? How about adding a docstring explaining what happens inside and why?